bin/ember

```diff
@@ -30,5 +30,7 @@ resolve('ember-cli', {
   }).then(function(result) {
     var exitCode = typeof result === 'object' ? result.exitCode : result;
     exit(exitCode);
+  }).catch(function() {
```

Comment:
the goal is for ember internals, to always resolve but with a status code.  This should is an invariant that is easy to maintain.